webhooks-extension/src/components/WebhookDisplayTable/tests/WebhookTable.test.js

```diff
@@ -24,22 +24,29 @@ describe('without webhooks', () => {
   const noWebhooks = [ {} ];
   it('should display Loading when loading', () => {
     jest.spyOn(API, 'getWebhooks').mockImplementation(() => Promise.resolve([noWebhooks]));
-    const { queryByText, queryByTestId } = renderWithRouter(<WebhookDisplayTable match={{}} />);
+    const { queryByText, queryByTestId } = renderWithRouter(<WebhookDisplayTable match={{}} namespace="*"/>);
     expect(queryByTestId('webhook-notification')).toBeNull();
     expect(queryByText(/Loading/i)).toBeTruthy();
   });
-}); 
+
+  it('should display string suggesting to create webhooks when no namespace has webhooks', () => {
+    const noWebhooks = [ ];
```

Comment:
there's no reference to this 'string suggesting to create webhooks' in the test, this looks identical to the test above

Reply:
oops must of not pushed that part

Reply:
removed after noticing that when there are no webhooks under any namespace the page redirects to the creating modal.